src/sentry/api/endpoints/project_ownership.py

```diff
@@ -162,6 +165,25 @@ def get_ownership(self, project):
                 last_updated=None,
             )
 
+    def add_owner_id_to_schema(self, ownership: ProjectOwnership, project: Project) -> None:
+        if ownership and (
+            not hasattr(ownership, "schema")
+            or "id" not in ownership.schema["rules"][0]["owners"][0].keys()
```

Comment:
does this break if there's currently no rules, like if they deleted all their rules

Reply:
thank you, good catch. I've added checks for this